spec/importer_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'spec_helper'
+require 'ronin/masscan/importer'
+require 'masscan/output_file'
+require 'ronin/db'
+
+RSpec.describe Ronin::Masscan::Importer do
+  let(:masscan_path) { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures', 'masscan.json')) }
```

Comment:
Also add a `let(:fixtures_dir) { ... }` to shorten that line to `File.join(fixtures_dir,'masscan.json')`.